app/(admin)/layout.tsx

```diff
@@ -1,12 +1,23 @@
 // Copyright (c) Gridiron Survivor.
 // Licensed under the MIT License.
 
-import React, { JSX } from 'react';
-import { GeistSans } from 'geist/font/sans';
 import '../globals.css';
 import { AuthContextProvider } from '@/context/AuthContextProvider';
-import ErrorBoundary from '../error';
+import { GeistSans } from 'geist/font/sans';
+import {
+  LucideBell,
+  LucideChevronsUpDown,
+  LucideCog,
+  LucideHome,
+  LucideLayoutGrid,
+  LucideUsers,
+} from 'lucide-react';
 import { Toaster } from 'react-hot-toast';
+import ErrorBoundary from '../error';
```

Comment:
Can you use `import ErrorBoundary from '@/error'` ?

Reply:
Good call, I'll do that.